pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -77,21 +79,17 @@ rustPlatform.buildRustPackage {
 
   buildInputs = runtimeDeps;
 
-  installPhase = "" + lib.optionalString stdenv.isLinux ''
+  preFixup = "" + lib.optionalString stdenv.isLinux ''
     for artifact in wezterm wezterm-gui wezterm-mux-server strip-ansi-escapes; do
-      patchelf --set-rpath "${lib.makeLibraryPath runtimeDeps}" $releaseDir/$artifact
-      install -D $releaseDir/$artifact -t $out/bin
+      patchelf --set-rpath "${lib.makeLibraryPath runtimeDeps}" $out/bin/$artifact
     done
   '' + lib.optionalString stdenv.isDarwin ''
-  mkdir -p "$out/Applications"
-  OUT_APP="$out/Applications/WezTerm.app"
-  cp -r assets/macos/WezTerm.app "$OUT_APP"
-  rm $OUT_APP/*.dylib
-  cp -r assets/shell-integration/* "$OUT_APP"
-  cp $releaseDir/wezterm "$OUT_APP"
-  cp $releaseDir/wezterm-mux-server "$OUT_APP"
-  cp $releaseDir/wezterm-gui "$OUT_APP"
-  cp $releaseDir/strip-ansi-escapes "$OUT_APP"
+    mkdir -p "$out/Applications"
+    OUT_APP="$out/Applications/WezTerm.app"
+    cp -r assets/macos/WezTerm.app "$OUT_APP"
+    rm $OUT_APP/*.dylib
+    cp -r assets/shell-integration/* "$OUT_APP"
+    ln -s $out/bin/{wezterm,wezterm-mux-server,wezterm-gui,strip-ansi-escapes} "$OUT_APP"
```

Comment:
It would be great if someone could test this part.

Reply:
```
results/wezterm/bin/wezterm
 2021-04-01T11:59:05.989Z INFO  wezterm_mux_server_impl::local > setting up /run/user/1000/wezterm/gui-sock-374667
 2021-04-01T11:59:05.990Z ERROR wezterm_gui                    > failed to open X11 display; terminating
 2021-04-01T11:59:05.991Z ERROR wezterm_toast_notification::dbus > while showing notification: org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files
 ```

I am to lazy to test this not headless but it looks already not quite right.

Reply:
If this is a headless machine without x11 than this looks good to me. I think using dbus on macOS is an upstream bug. There is no dbus-based notifications on macOS.

Reply:
i've got a build failure, https://gist.github.com/cransom/db7fe99aeec946ca216192a071059db0

Reply:
I added libiconv. Does this fix the build for you?